### ISSUE TITLE
Better successful build message

### ISSUE THREAD

**Issue by kommen** [deadfoxygrandpa/Elm.tmLanguage#89]
When my build code compiles without errors I see the following message at the top:

```
=== Successfully generated /dev/null ===
[Finished in 0.4s]
```

I know this [reason][tweet] for this. But would it be possible to replace the `generated /dev/null` to something simpler? I think that the `/dev/null` build output should be an implementation detail of which the user should not be concerned with.

[tweet]: https://twitter.com/rtfeldman/status/624026168652660740

**Comment by deadfoxygrandpa** [comment]
I agree that this would be nicer. I'll try to get something in the next version.

**Comment by msjyoo** [comment]
An even better option would be to use a plugin like this one https://packagecontrol.io/packages/Better%20Build%20System to hide the panel on successful builds.
